src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -500,19 +497,20 @@ private SmugglerProgressBase.Counts ProcessDocuments(SmugglerResult result, Buil
                 {
                     _token.ThrowIfCancellationRequested();
                     result.Documents.ReadCount++;
-
-                    if (result.Documents.ReadCount % 1000 == 0)
+                    var totalCount = result.Documents.ReadCount - result.Documents.SkippedCount - result.Documents.ErroredCount;
```

Comment:
We do not want this. ReadCount is enough.

Reply:
done